plugins.go

```diff
@@ -224,10 +225,24 @@ func (d *DynamicMiddleware) ProcessRequest(w http.ResponseWriter, r *http.Reques
 		"prefix": "jsvm",
 	}).Debug("JSVM middleware execution took: (ns) ", time.Now().UnixNano()-t1)
 
-	if newRequestData.Request.ReturnOverrides.ResponseCode != 0 {
+	if newRequestData.Request.ReturnOverrides.ResponseCode != 0 && newRequestData.Request.ReturnOverrides.ResponseCode > 300 {
```

Comment:
all values `> 300` are `!= 0`, so the first half can go.

Also, if you meant to do this for all error status codes, you probably want `>= 400`. Otherwise, I'm not sure why the `> 300` was chosen.